11/bullseye/Dockerfile

```diff
@@ -44,10 +44,11 @@ ENV JENKINS_AGENT_HOME ${JENKINS_AGENT_HOME}
 RUN groupadd -g ${gid} ${group} \
     && useradd -d "${JENKINS_AGENT_HOME}" -u "${uid}" -g "${gid}" -m -s /bin/bash "${user}"
 
-# setup SSH server
 RUN apt-get update \
-    && apt-get install --no-install-recommends -y openssh-server \
+    && apt-get install --no-install-recommends -y openssh-server git-lfs \
```

Comment:
Shouldn't this be `git` instead of `git-lfs`?
Is `git-lfs` needed/wanted?

Reply:
its included in the other images fine for consistency.